_dev/src/components/product-feed/product-feed-card.vue

```diff
@@ -186,29 +186,33 @@
                 :title="$t('productFeedSettings.shipping.targetCountries')"
                 :description="targetCountries.join(', ')"
                 :link="$t('cta.editCountries')"
-                :link-to="{ type: 'routeStep', name: 'product-feed-settings', step: 1 }"
+                :link-to="{ type: 'routeStep', name: 'product-feed-settings',
+                            step: 1, params: ProductFeedSettingsPages.TARGET_COUNTRY }"
```

Comment:
Did you mean

```suggestion
                            step:  ProductFeedSettingsPages.TARGET_COUNTRY }"
```

?